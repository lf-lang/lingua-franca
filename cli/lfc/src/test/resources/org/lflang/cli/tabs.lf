target C;

main reactor {
	state foo = 1;
	state	bar = 1;
}
