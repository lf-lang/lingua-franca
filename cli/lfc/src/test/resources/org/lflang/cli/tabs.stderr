lfc: error: State must have a type.
--> %%%PATH.lf%%%:4:2
  |
3 | main reactor {
4 |     state foo = 1;
  |     ^^^^^^^^^^^^^^ State must have a type.
  |
5 |     state    bar = 1;
lfc: error: State must have a type.
--> %%%PATH.lf%%%:5:2
  |
4 |     state foo = 1;
5 |     state    bar = 1;
  |     ^^^^^^^^^^^^^^^^^ State must have a type.
  |
6 | }